### PR TITLE
Handled the deprecated secant_latitudes keyword in cartopy.

### PR DESCRIPTION
**Pull request by pelson**

### CODE REVIEW

**Comment by rhattersley** [comment]
:+1: ... just needs the copyright period adjusting

**Comment by pelson** [comment]
I think @marqh is interested in this one...

File: lib/iris/coord_systems.py [hunk 1]
```diff
@@ -23,7 +23,8 @@
 from abc import ABCMeta, abstractmethod
 import warnings
 
-import cartopy.crs
+import cartopy
+import cartopy.crs as ccrs
```
Comment (QuLogic) [inline lib/iris/coord_systems.py]: I could've sworn I had a PR that fixed all of these...

Comment (pelson) [inline lib/iris/coord_systems.py]: Against master maybe?

**Comment by bjlittle** [comment]
:+1: Awesome, thanks @pelson

**Comment by pelson** [comment]
Merged to master with aa6e233.

File: lib/iris/coord_systems.py (hunk 1 above)
Comment (QuLogic) [inline lib/iris/coord_systems.py]: Ah, that might be it.

Comment (ajdawson) [inline lib/iris/coord_systems.py]: #1376 (against master)

**Pull request merged**